proxy/server.py

```diff
@@ -596,7 +597,10 @@ def do_GET(self):
                 'soe': pw.level() or 0,
                 'grid_status': int(pw.grid_status() == 'UP'),
                 'reserve': pw.get_reserve() or 0,
-		'time_remaining_hours': pw.get_time_remaining() or 0
+                'time_remaining_hours': pw.get_time_remaining() or 0,
+                'full_pack_energy': get_value(d, 'nominal_full_pack_energy') or 0,
+                'energy_remaining': get_value(d, 'nominal_energy_remaining') or 0,
```

Comment:
I'm on the fence about truncating this value.  

```suggestion
                'energy_remaining': math.trunc(get_value(d, 'nominal_energy_remaining')) or 0,
```

1. I like reporting the value you get;
1. The actual value that is returned in my case has a 13 digit decimal.  The unit here is Watt-Hours, so there's little significance in reporting a 4x10^-13 Watt-hours... I did the math: that's 14 milliwatt-µseconds, hah.

Reply:
```json
  "full_pack_energy": 42250,
  "energy_remaining": 30200.000000000004,
  ```

Reply:
In my opinion, the proxy should output exactly the values as they come from the TEDAPI. 
Further processing, e.g. rounding, is normally the task of a downstream system. 
@jasonacox What is your opinion on this?

Reply:
Yeah, after sleeping on it, seems like a post-processing thing to drop the decimal.